.editorconfig

```diff
@@ -1,8 +0,0 @@
-# 4 space indentation
-[*.cs]
-indent_style = space
-indent_size = 4
-
-[*.ps1]
-indent_style = space
-indent_size = 4
```

Comment:
Is there a reason for removing this?

Reply:
I overlooked that I had inadvertently removed this - this is added back now.